client/src/components/FillChart.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <div>
+    <apexchart
+      height="400"
+      width="100%"
+      :options="chartOptions"
+      :series="series"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "FillChart",
+  props: ["data"],
+  data() {
+    return {
+      dateRange: 30,
+      chartOptions: {
+        chart: {
+          type: "area",
+          stacked: false,
+          width: "auto",
+          toolbar: {
+            autoSelected: "zoom",
+            show: true,
+          },
+          dropShadow: {
+            enabled: true,
+            enabledSeries: [0],
+            top: -2,
+            left: 2,
+            blur: 5,
+            opacity: 0.06,
+          },
+        },
+        colors: ["#accc20", "#1aa5ca", "#f77046", "#ADB5BD", "#AF125A"],
```

Comment:
Might be a good idea to add these as graph color variables in the scss variables file